build.gradle

```diff
@@ -39,9 +39,8 @@ dependencies {
 	annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
 	annotationProcessor 'org.projectlombok:lombok'
 	providedRuntime 'org.springframework.boot:spring-boot-starter-tomcat'
-	testImplementation('org.springframework.boot:spring-boot-starter-test') {
-		exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
-	}
+	testImplementation 'org.springframework.boot:spring-boot-starter-test:2.2.6.RELEASE'
```

Comment:
should we have version here? if then it should be 2.3.1.RELEASE

Reply:
No, I'll remove the version. But the target branch is wrong, so I'll delete this branch and push another one.